examples/cookbook/lists/floating_app_bar/lib/starter_cupertino.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/cupertino.dart';
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    // #docregion CustomScrollView
+    return const CupertinoApp(
+      home: CupertinoPageScaffold(
```

Comment:
The Material starter excludes the `MaterialApp` widget, should we exclude the `CupertinoApp` widget as well?

![image](https://github.com/user-attachments/assets/5fc05722-15c6-4c64-ac3a-c78a9a36c452)

Reply:
Thank you for catching this. I've included the `MaterialApp` widget in the material example.

Reply:
Sounds good, closing!